docs/api/security/crypto.md

```diff
@@ -0,0 +1,49 @@
+## Crypto
+
+Arm Mbed Crypto is the reference implementation of the cryptography interface
+of the Arm Platform Security Architecture (PSA). This is a preview release of
+Mbed Crypto, provided for evaluation purposes only.
+
+We have adapted and [integrated Mbed Crypto with Mbed
+OS](https://github.com/ARMmbed/mbed-os/blob/master/features/mbedtls/mbedcrypto).
+On PSA platforms that support it, Mbed Crypto is pre-integrated with Mbed OS to
+take advantage of the platform's segmented architecture and isolate
```

Comment:
"take advantage of" has a negative connotation to me. What do you think about replacing it with "make the most of" or "use"?

Reply:
I've tried "leverage".